services/AUTOMATIC1111/entrypoint.sh

```diff
@@ -57,9 +57,9 @@ chown -R root ~/.cache/
 chmod 766 ~/.cache/
 
 shopt -s nullglob
-list=(./extensions/*/requirements.txt)
-for req in "${list[@]}"; do
-  pip install -r "$req"
+list=(./extensions/*/install.py)
+for installscript in "${list[@]}"; do
+  PYTHONPATH=${ROOT} python "$installscript"
```

Comment:
is the `PYTHONPATH` necessary?

Reply:
oh I see, it is here: https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Developing-extensions#installpy

Would be great if you could add this as a comment

Reply:
Yes, from the results of my test, `PYTHONPATH` is necessary, otherwise some modules provided by the webui project will not be found correctly